charmcraft/commands/init.py

```diff
@@ -35,6 +35,7 @@
     "simple": "init-simple",
     "kubernetes": "init-kubernetes",
     "machine": "init-machine",
+    "flask-framework": "init-flask-framework",
```

Comment:
Please update `_overview` below to include a description of this template.

Reply:
I have added an one sentence summary for now, but I will add a link to the document once we have published the document for the 12-factor project.